apps/user-office-frontend/src/components/App.tsx

```diff
@@ -107,36 +107,53 @@ class App extends React.Component {
 
   render(): JSX.Element {
     return (
-      <StyledEngineProvider injectFirst>
-        <SnackbarProvider
-          ref={this.notistackRef}
-          anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
-          maxSnack={1}
-          action={(key) => (
-            <IconButton onClick={this.onClickDismiss(key)}>
-              <Close htmlColor="white" />
-            </IconButton>
-          )}
-        >
-          <SettingsContextProvider>
-            <Theme>
-              <FeatureContextProvider>
-                <UserContextProvider>
-                  <DownloadContextProvider>
-                    <IdleContextPicker>
-                      <Router>
-                        <QueryParamProvider ReactRouterRoute={Route}>
-                          <Routes />
-                        </QueryParamProvider>
-                      </Router>
-                    </IdleContextPicker>
-                  </DownloadContextProvider>
-                </UserContextProvider>
-              </FeatureContextProvider>
-            </Theme>
-          </SettingsContextProvider>
-        </SnackbarProvider>
-      </StyledEngineProvider>
+      <Suspense
```

Comment:
I am just wondering if this is really needed because I think we already have some loader included in the main page or maybe I am wrong. Or at least we have some loading text when we load settings and features. If this works better then we can remove that one over there.

Reply:
I couldn't get i18n working without this, I get errors around a suspended while rendering, with no fall back UI specified. Looks like the loading text around setting and features is around waiting for API calls to the backend to be resolved so not a loader as such. Not sure if we have a loader else where can't seem to find one and I guess it would be too low down the react tree for i18n to use.

Reply:
OK then I understand we can keep it like that for now. We might need to try to remove the other loaders if they are not needed but that is a task for another day maybe. Thanks for the explanation.